parachain/pallets/commitments/src/lib.rs

```diff
@@ -133,14 +137,13 @@ impl<T: Config> Module<T> {
 
 		ethabi::encode(&vec![Token::FixedArray(messages)])
 	}
-
 }
 
 impl<T: Config> MessageCommitment for Module<T> {
 
 	// Add a message for eventual inclusion in a commitment
 	// TODO: Number of messages per commitment should be bounded
```

Comment:
can we move this todo to a github issue flagged with Security?

Reply:
Sure, added #226.